serverless/admin_guide/serverless-cluster-admin-eventing.adoc

```diff
@@ -1,12 +1,12 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="serverless-cluster-admin-eventing"]
-= Creating Eventing components as a cluster administrator
+= Creating Eventing components in the Administrator perspective
```

Comment:
Do you want to update the topic map with the new title?

Reply:
Also, should you also make this change for the _Creating Serving components as a cluster administrator_ topic?

Reply:
Updated for this and serving, thanks